src/reason_pprint_ast.ml

```diff
@@ -2561,16 +2561,17 @@ class printer  ()= object(self:'self)
       match (x.ptyp_desc) with
         | (Ptyp_arrow (l, ct1, ct2)) ->
           let rec allArrowSegments acc = function
-            | { ptyp_desc = Ptyp_arrow (l, ct1, ct2); _ } ->
+            | { ptyp_desc = Ptyp_arrow (l, ct1, ct2); ptyp_attributes } when List.length ptyp_attributes == 0 ->
```

Comment:
Would this be better? `ptyp_attributes = []`

Reply:
updated